backend/src/errors.rs

```diff
@@ -3,89 +3,94 @@ use axum::{
     response::{IntoResponse, Json, Response},
 };
 use serde_json::json;
-use sqlx::Error as SqlxError;
-use std::fmt;
+use sqlx::{error::DatabaseError, Error as SqlxError};
 
 pub enum CustomError {
-    NotFound(String),
```

Comment:
How are we not using this?

Reply:
My thinking was that NotFound is covered in BadRequest

Reply:
[3d1d7a8](https://github.com/LukeBrandon/fluke/pull/57/commits/3d1d7a8d65afa3e1027e9e7e36ea515a77d99d0f)